README.md

```diff
@@ -658,6 +659,59 @@ Tests whether some element in the array passes the test implemented by the provi
 
 **[⬆ back to top](#quick-links)**
 
+### _.minBy and _.maxBy
+
+Use Array#reduce for find the maximum or minimum collection item
+
+  ```js
+  // Underscore/Lodash
+  var data = [{ value: 6 }, { value: 2 }, { value: 4 }]
+  var minItem = _.minBy(data, 'value')
+  var maxItem = _.maxBy(data, 'value')
+  console.log(minItem, maxItem)
+  // output: { value: 2 } { value: 6 }
+
+  // Native
+  var data = [{ value: 6 }, { value: 2 }, { value: 4 }]
+  var minItem = data.reduce(function(a, b) { return a.value <= b.value ? a : b }, {})
+  var maxItem = data.reduce(function(a, b) { return a.value >= b.value ? a : b }, {})
+  console.log(minItem, maxItem)
+  // output: { value: 2 }, { value: 6 }
+  ```
+
+Extract a functor and use es2015 for better code
+
+  ```js
+  // utils
+  const makeSelect = (comparator) => (a, b) => comparator(a, b) ? a : b
+  const minByValue = makeSelect((a, b) => a.value <= b.value)
+  const maxByValue = makeSelect((a, b) => a.value <= b.value)
+
+  // main logic
+  const data = [{ value: 6 }, { value: 2 }, { value: 4 }]
+  const minItem = data.reduce(minByValue, {})
+  const maxItem = data.reduce(minByValue, {})
```

Comment:
seems a typo here...

 should we create a ESlint rule for _.minBy _.maxBy after this merge? @stevemao